driver-core/src/test/unit/com/mongodb/MongoCredentialSpecification.groovy

```diff
@@ -27,37 +27,39 @@ import static com.mongodb.AuthenticationMechanism.SCRAM_SHA_1
 class MongoCredentialSpecification extends Specification {
     def 'creating a credential with an unspecified mechanism should populate correct fields'() {
         given:
-        String userName = 'user';
-        String database = 'test';
-        char[] password = 'pwd'.toCharArray();
+        String userName = 'user'
```

Comment:
The formatting changes are a bit distracting from the main purpose. Can you revert and commit separately?